### ISSUE TITLE
Bflabel issue

### ISSUE THREAD

**Issue by Lordmzn** [Lordmzn/ClassicThesis-at-DEIB#1]
```
\bflabel undefined.See the LaTeX manual or LaTeX companion....
```

Encountered on windows + miktex

**Comment by Lordmzn** [comment]
Solution: [look here](http://tex.stackexchange.com/questions/207833/bflabel-undefined)

**Comment by Lordmzn** [comment]
Or better solution: https://code.google.com/p/classicthesis/issues/detail?id=95

Fixed with commit e4bab6910a367ea78ae539716cb8a27c0e29a906
